dbt/project.py

```diff
@@ -2,8 +2,10 @@
 import yaml
 import pprint
 import copy
+import sys
 
 default_project_cfg = {
+    'package': {'name': None, 'version': None},
```

Comment:
Is this necessary?  If we're going to specify something here, I think it should at least pass our `validate` check.